relay_messages.go

```diff
@@ -167,6 +172,19 @@ func (f lazyCallReq) TTL() time.Duration {
 	return time.Duration(ttl) * time.Millisecond
 }
 
+func (f lazyCallReq) SetTTL(d time.Duration) error {
+	if d < 0 {
+		return errTTLNegative
+	}
+	millis := d / time.Millisecond
+	if millis > math.MaxUint32 {
+		return errTTLOverflow
```

Comment:
these error cases can be caught at channel creation time since it points to a strange configuration error. Should we instead do this check once when creating the `Relayer`, log errors, and then store the `maxTimeTTL` which is either the user-configured value, or 2 mins if not configured or the value passed in is invalid.

Once we do that, this method doesn't need to return an error which simplifies some of the caller logic and removes logging in this path.